File: 백준/Bronze/25640. MBTI/MBTI.py
import sys

input = sys.stdin.readline

jinho = input().strip()
N = int(input().strip())
result = 0

for _ in range(N):
    mbti = input().strip()
    if mbti == jinho:
        result += 1

sys.stdout.write(str(result) + '\n')

File: 백준/Bronze/25640. MBTI/MBTI.java
import java.io.*;

public class Main{
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		
        String jinho = br.readLine().trim();
        int N = Integer.parseInt(br.readLine().trim());
        int result = 0;

        for(int i = 0; i < N; i++){
            String mbti = br.readLine().trim();
            if(mbti.equals(jinho)) result++;
        }

		bw.write(String.valueOf(result) + '\n');
		bw.flush();
		bw.close();
		br.close();
	}
}